backend/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+
+/** A user with saved preferences. */
+public final class User {
+
+  /** Datastore ID */
+  private final long id;
+  /** Google username (ldap) */
+  private final String username;
+  /** Time of submitted form */
+  private final long timestamp;
```

Comment:
A user who chooses to save preferences with ad-lib will have multiple form timestamps. Does this make sense on this model?